sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -513,8 +514,10 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       isDataSourceTable: Boolean,
       format: String,
       userSpecifiedLocation: Option[String] = None): Unit = {
-    val relation = EliminateSubqueryAliases(
-      sessionState.catalog.lookupRelation(TableIdentifier(tableName)))
+    var relation: LogicalPlan = null
+    withSQLConf(HiveUtils.CONVERT_METASTORE_PARQUET.key -> "false") {
```

Comment:
I checked the test cases. `ORC` has the same issue, but the default value is `false` currently. Thus, I think we should set `CONVERT_METASTORE_ORC` to `false` too, in case we change the default value of `CONVERT_METASTORE_ORC` in the future.